app/webpacker/components/CompetitionForm/FormSections/RegistrationDates.js

```diff
@@ -32,6 +38,7 @@ export default function RegistrationDates() {
           startDate={openingDateTime}
           endDate={closingDateTime}
           minDate={openingDateTime}
+          ignoreDisabled={registrationNotYetPast}
```

Comment:
Running locally on WashtenawFMC22025, the Registration Close date remains un-editable: 
![image](https://github.com/user-attachments/assets/02a21d3c-add8-44b3-b7e5-7482d1a0a7d3)

Reply:
Fixed!